internal/pkg/sysinfo/sysinfo.go

```diff
@@ -52,7 +52,7 @@ func (s *K0sSysinfoSpec) RunPreFlightChecks(lenient bool) error {
 }
 
 func (s *K0sSysinfoSpec) NewSysinfoProbes() probes.Probes {
-	p := probes.NewProbes()
+	p := probes.NewProbes(nil)
```

Comment:
Looks a bit fishy, is it on purpose?

Reply:
IMHO, both will be better than `nil`, it's up to you what to choose :)

Reply:
Done.